content/crosswalk/aws.md

```diff
@@ -1,4 +1,8 @@
 ---
 title: Modern Infrastructure as Code for AWS with Pulumi
 layout: aws
+
+meta_title: Well-Architected Infrastructure as Code for AWS
```

Comment:
This is maybe a question for @joeduffy; I can imagine it might've been intentional to specify this particular string in social contexts, since it's a little more specific than what shows up in the hero title.